### PR TITLE
feat(Discord ID): add presence

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added presence for Discord.id

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by nerrixDE** [comment]
privacy concerns
~ Owner of discord.id

**Comment by DarkVillager** [comment]
> privacy concerns ~ Owner of discord.id
New default:
image

**Comment by nerrixDE** [comment]
good

File: websites/D/Discord ID/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Discord ID",
+	"description": {
+		"en": "Lookup for all type of discord ids."
```
Comment (EncryptedDev) [inline websites/D/Discord ID/dist/metadata.json:9]:
```suggestion
		"en": "Lookup a Discord User or Bot ID."
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**